client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/translations.tsx

```diff
@@ -25,6 +30,12 @@ export function getLaunchpadTranslations( flow: string | null ): TranslatedLaunc
 				"All that's left is to add some links and launch your site."
 			);
 			break;
+		case FREE_FLOW:
```

Comment:
You can add these to the translations specs

Reply:
Added to specs!